ark/mibi/tiling_utils.py

```diff
@@ -1150,15 +1102,36 @@ def remap_and_reorder_tiles(tiled_regions_proposed, proposed_to_auto_map,
     if moly_interval <= 0:
         raise ValueError("moly_interval must be at least 1")
 
-    # rename the FOVs based on the mapping
+    # define a new tiled regions proposed dict
+    tiled_regions_remapped = {}
+
+    # if the user specifies additional metadata, make sure they're copied over
+    # specifically, id, name, and status
+    if 'id' in tiled_regions_proposed:
+        tiled_regions_remapped['id'] = tiled_regions_proposed['id']
```

Comment:
We should have a helper function that copies over whatever the metadata is at the top of the JSON instead of hard coding it. So anything that's just a single string.

Reply:
To clarify, does this mean it's OK to leave out `dict` fields such as `opticalImage` if specified, or should we copy those over too?

Reply:
Hmm, can't hurt to include those I guess. It's not needed for loading